### PR TITLE
webpack watches for changes

### PR DESCRIPTION
**Pull request by michael-jacques**
This PR:

- lets webpack listen for changes to js(x) files 

why?

So we dont have to restart webpack every time we make changes

### CODE REVIEW

**Comment by pachun** [comment]
👍 , ship it. Keep deleting code, please.

File: package.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
     "build": "webpack",
     "lint": "eslint src",
     "mocha": "NODE_PATH=./src mocha --compilers babel-core/register --require src/test/.setup.js --recursive './{,!(node_modules)/**/}*.tests.js*'",
-    "start": "npm run build && node server.js",
+    "start:dev": "webpack --watch",
+    "start:prod": "npm run build",
```
Comment (zfletch) [inline package.json]: Do we not need `&& node server.js` like in the previous start command?

Comment (mikestone14) [reply package.json]: I don't think so since this serves the static html file which uses the bundled js and css.

Comment (mikestone14) [reply package.json]: The `server.js` file is deleted as part of this PR.

File: package.json [hunk 2]
```diff
@@ -10,7 +10,8 @@
     "build": "webpack",
     "lint": "eslint src",
     "mocha": "NODE_PATH=./src mocha --compilers babel-core/register --require src/test/.setup.js --recursive './{,!(node_modules)/**/}*.tests.js*'",
-    "start": "npm run build && node server.js",
+    "start:dev": "npm run build && node server.js",
+    "start:prod": "npm run build && node server.js",
```
Comment (taylorkearns) [inline package.json]: Do we need the two methods if they execute the same thing?

**Comment by taylorkearns** [comment]
@michael-jacques nice!

**Pull request merged**